.github/workflows/build-frontend-canister.yml

```diff
@@ -0,0 +1,32 @@
+name: Check frontend canister build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up
+        run: |
+          rustup target add wasm32-unknown-unknown
+          cargo install --version 0.1.5 ic-wasm
+      - name: Build frontend canister
+        run: |
+          ./scripts/update-frontend-canister.sh
+      - name: Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: assetstorage
+          path: ${{ github.workspace }}/src/distributed/assetstorage.wasm.gz
```

Comment:
What's the purpose of uploading this an an artifact here?  It's not used in a later step

Reply:
The purpose is to allow the developer to check in the artifact in case the test fails (alternatively, the developer would need to run `./scripts/update-frontend-canister.sh`).